### ISSUE TITLE
Empty annotation file on S3

### ISSUE THREAD

**Issue by berguner** [Novartis/cellxgene-gateway#61]
Hi,

I am running the gateway with cellxgene==1.0.0 on an AWS EC2 instance using docker. If I open the .h5ad files on S3 storage, the annotation and gene-set files are created but they aren't filled in. The EC2 instance has full read/write access to the S3 bucket. I also keep getting a 500 error every 2-3 seconds once I start annotating the data:

```
2022-02-15 15:11:58,004:werkzeug:INFO:10.252.248.24 - - [15/Feb/2022 15:11:58] "PUT /source/s3/view/Human_PBMC_HO_aggr_annotations/berguner.csv/api/v0.2/annotations/obs?annotation-collection-name=berguner HTTP/1.1" 500 -
```

This doesn't happen while working on local data. Do you have any idea why this might happen? How can I debug this?

**Comment by arogozhnikov** [comment]
I think you can find this PR too cellxgene very relevant, 
https://github.com/chanzuckerberg/cellxgene/pull/2501

There are some issues on gateway side as well

**Comment by alokito** [comment]
@berguner As @arogozhnikov this should be fixed by upgrading to [Cellxgene v 1.1.0](https://github.com/chanzuckerberg/cellxgene/releases/tag/1.1.0), please comment if this issue is still relevant, otherwise I will close next weekend.

**Comment by arogozhnikov** [comment]
@alokito I don't have relevant production deployment to test, so I'm fine with closing
